<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRUD</class>
 <widget class="QMainWindow" name="CRUD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRUD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="botaoExport">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoImport">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoRemove">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linhaLabel">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>81</y>
      <width>125</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>61</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoAdd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>117</y>
      <width>256</width>
      <height>186</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Label</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="linhaID">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>32</y>
      <width>125</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>12</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
